MonoGame.Framework/Graphics/Vertices/VertexBuffer.cs

```diff
@@ -54,13 +54,18 @@ internal protected override void GraphicsDeviceResetting()
 
         public void GetData<T> (int offsetInBytes, T[] data, int startIndex, int elementCount, int vertexStride) where T : struct
         {
+            var elementSizeInBytes = Marshal.SizeOf(typeof(T));
+
+            if (vertexStride == 0)
+                vertexStride = elementSizeInBytes;

```

Comment:
This is good information for docs on the method.  Stuff that XNA doesn't even mention.